Dockerfile

```diff
@@ -0,0 +1,37 @@
+FROM centos:centos7
+
+EXPOSE 8080
+
+ADD sendEmail-epel-7.repo /etc/yum.repos.d/
+
+RUN yum install -y epel-release && \
+    yum -y --setopt=tsflags=nodocs update && \
+    yum -y --setopt=tsflags=nodocs install httpd \
+                                           nagios \
+                                           telnet \
+                                           supervisor \
+                                           python-jinja2 \
+                                           nagios-plugins-all \
+                                           sendEmail \
+                                           perl-Net-SSLeay \
+                                           perl-IO-Socket-SSL && \
+    yum clean all && \
+    mkdir -p /opt/rhmap/ && \
+    sed -i -e 's/Listen 80/Listen 8080/' /etc/httpd/conf/httpd.conf && \
+    sed -i -e 's|DocumentRoot "/var/www/html"|DocumentRoot "/usr/share/nagios/html"|' -e 's|<Directory "/var/www">|<Directory "/usr/share/nagios/html">|' /etc/httpd/conf/httpd.conf && \
+    touch /supervisord.log supervisord.pid && \
+    mkdir -p /var/log/nagios/rw/ && \
+    mkdir -p /var/log/nagios/spool && \
+    mkdir -p /var/log/nagios/spool/checkresults && \
+    chmod -R 777 /supervisord.log /supervisord.pid /var/log/nagios \
+                    /etc/httpd /etc/passwd /var/log /etc/nagios /usr/lib64/nagios /var/spool/nagios /run /usr/share/httpd /usr/share/nagios && \
```

Comment:
The indentation here is strange.

Perhaps a comment above the `chmod` explaining why we need to loosen permissions for those particular files/dirs wouldn't do bad.

Not obvious, Dockerfile comments work just fine in the middle of RUN instructions. E.g.: https://github.com/sclorg/mongodb-container/blob/master/3.2/Dockerfile#L33